Jellyfin.Api/Auth/SyncPlayAccessPolicy/SyncPlayAccessHandler.cs

```diff
@@ -47,6 +48,10 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
 
             var userId = context.User.GetUserId();
             var user = _userManager.GetUserById(userId);
+            if (user is null)
```

Comment:
If this happens, it's either an api key or the user was deleted between the authentication and authorization steps

Reply:
What would you do here to prevent a nullref exception a bit lower?